### ISSUE TITLE
How to better manage secrets?

### ISSUE THREAD

**Issue by adborden** [openoakland/infra#13]
In theory, we don't need to store secrets (e.g. database passwords) out-of-band from the project (i.e. avoid having to share secrets through 1password). Any user with AWS credentials with access to the S3 bucket and the Terraform state file should be able to pull the secrets using `terraform output`. Once we figure out a more precise workflow, we should document it.

**Comment by tdooner** [comment]
Perhaps the task here is to improve documentation of getting set up with the AWS credentials necessary to run Terraform to get these secrets?

**Comment by adborden** [comment]
Agreed,I think based on Rik's testing on slack, I think we have a start.
